### PR TITLE
Tw/moderator guide

### PR DESCRIPTION
**Pull request by shereefb**
Game over conditions and very rough moderator guide

@tannerwelsh please review and merge

### CODE REVIEW

File: Game_Manual/Boundaries.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+## Objectives
+
+The game is designed to support players in tracking and achieving two objectives:
+
+1. To become a world class beginner in their craft
+1. To co-create a culture of collective learning
+
+## Success Conditions
+
+A player succeeds when they (a) meet the above objectives and (b) fulfill the obligations in their contract.
```
Comment (tannerwelsh) [inline Game_Manual/Boundaries.md:16]: Is this accurate / clear?

File: Game_Manual/Boundaries.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+Fail conditions are triggered when any of the following persists over multiple cycles:
+
+1. [Time Dedication per Cycle][time-dedication] is too low
+1. Rate of [XP][xp] growth is too low
+1. [Learning Support][learning-support] is too low
+1. [Culture Contribution][culture-contribution] is too low
+1. [Discernment][discernment] is too high or too low
+
+If these conditions are met, the Moderator will expel the player from the game.
```
Comment (tannerwelsh) [inline Game_Manual/Boundaries.md]: I can't figure out what word is best for this. I don't like "kick out" (kinda physically violent) or "expel" (too school-related), but I prefer expel for now.

**Comment by tannerwelsh** [comment]
@shereefb I made some non-trivial changes that I want your eyes on before merging. If you're ok with them, feel free to merge.

Highlights:
- rename Assessment Accuracy -> Discernment
- rename Average Hours per Week -> Time Dedication per Cycle (IMHO we shouldn't be looking at average hours, but consistently low hours in multiple cycles)
- changed stat representation for Learning Support and Culture Contribution (percent 0..100% not 1..7, see comment https://github.com/LearnersGuild/game/issues/111#issuecomment-236168664 for rationale)
- made the fail conditions trigger when one "persists over multiple cycles"

**Comment by shereefb** [comment]
@tannerwelsh changes look good. I really like them. 

Switched TD per cycle, to TD per week. And removed it from fail conditions! I think it's rolled up in XP and we don't need it. We can always add it later.

**Pull request merged**